.vscode/launch.json

```diff
@@ -15,7 +15,10 @@
             "cwd": "${workspaceFolder}",
             // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console
```

Comment:
Comments are nonstandard in JSON, please consider removing

Reply:
I'm going to go on record to say that vscode added those, not me lol

Reply:
Excuses, excuses! :)